src/ui/components/map/providers/mapprovider.js

```diff
@@ -90,6 +90,25 @@ export default class MapProvider {
      * @type {boolean}
      */
     this._collapsePins = config.collapsePins || false;
+
+    /**
+     * Locale of the map. Defaults to the locale specified in ANSWERS.init() or to 'en'
+     * if a locale is not provided in the init.
+     * @type {string}
+     */
+    const locale = config.locale || config.initLocale || 'en';
+
```

Comment:
I'm getting the locale that ANSWERS was initialized from global storage and passing it here as the map component constructs the map provider. Is it preferable to access global storage inside this class rather than passing it through the component?

Reply:
The `MapProvider` doesn't have access to `globalStorage`. I think adding that in would be a bit cumbersome. I think it might simplify things to have this class take in only `config.locale` (not `config.initLocale`) and have the `MapComponent` be responsible for providing the right one. Potentially, the `MapComponent` can have a `_locale` member variable that's `config.locale || globalStorageValue` and that's passed to the provider as `config.locale`. Also, I think we can get rid of the `en` default as `globalStorage` should always have `LOCALE` stored.

Reply:
I like that solution. Updating.

Reply:
Oops. I deleted Tom's comment because it looks like it was duplicated, but it actually deleted the original comment as well. Here's what it said:

The MapProvider doesn't have access to `globalStorage`. I think adding that in would be a bit cumbersome. I think it might simplify things to have this class take in only `config.locale` (not `config.initLocale`) and have the MapComponent be responsible for providing the right one. Potentially, the MapComponent can have a `_locale` member variable that's `config.locale || globalStorageValue` and that's passed to the provider as `config.locale`. Also, I think we can get rid of the en default as `globalStorage` should always have `LOCALE` stored.

Reply:
Instead of a `_locale` member, I'm going to use Obect.assign() when creating the config to pass because the MapComponent doesn't seem to have access to the locale in the constructor.